src/onegov/form/validators.py

```diff
@@ -114,13 +115,88 @@ def __call__(self, form: Form, field: Field) -> None:
         if not field.data:
             return
 
-        if field.data.get('size', 0) > self.max_bytes:
+        if isinstance(field.data, list):  # UploadMultipleField
+            for data in field.data:
+                if not data:
+                    continue  # in case of file deletion
+
+                self.validate_filesize(field, data)
+
+        else:
+            self.validate_filesize(field, field.data)
+
+    def validate_filesize(self, field: Field, data: dict[Any, Any]) -> None:
+        if data.get('size', 0) > self.max_bytes:
             message = field.gettext(self.message).format(
                 humanize.naturalsize(self.max_bytes)
             )
             raise ValidationError(message)
 
 
+MIME_TYPES_PDF = {
+    'application/pdf',
+}
+
+# for now not allowed by default
+MIME_TYPES_JSON = {
+    'application/json',
+}
+
+MIME_TYPES_DOCUMENT = {
+    'application/msword',  # doc
+    'application/rtf',
+    *MIME_TYPES_PDF,
+    'application/vnd.ms-excel',  # xls
+    ('application/vnd.openxmlformats-officedocument.'
+     'presentationml.presentation'),  # pptx
+    ('application/vnd.openxmlformats-officedocument.'
+     'spreadsheetml.sheet'),  # xlsx
+    ('application/vnd.openxmlformats-officedocument.'
+     'wordprocessingml.document'),  # docx
```

Comment:
We might need to allow some extra mimetypes like `application/CDFV2` and `application/CDFV2-unknown`, which I believe can be reported for some really old Word files and `application/x-ole-storage` which can be reported for some old Excel files, especially since we did have some of those.

You could try to take a look at some example files, in order to verify that these are indeed legitimate files. loxo has some files with those mimetypes, so it should be fairly quick to check those three instances